### PR TITLE
Update NEWS.

### PR DESCRIPTION
**Pull request by mpreiner**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -2,20 +2,36 @@ This file contains a summary of important user-visible changes.
...
+
+- API: Added argument `fresh` to `Solver::declareSort` analogous to
+  `Solver::declareFun`.The option `--fresh-declarations` determines whether the
+  parser constructs fresh sorts for each declaration (true by default, which
+  matches the previous behavior).
+
+cvc5 1.0.7
+==========
+
 - API: C++ enums are now enum classes
```
Comment (aniemetz) [inline NEWS.md:17]: This shouldn't have been in 1.0.7, was it really?

File: NEWS.md [hunk 2]
```diff
@@ -2,20 +2,36 @@ This file contains a summary of important user-visible changes.
...
+- API: Added argument `fresh` to `Solver::declareFun` which distinguishes whether
```
Comment (aniemetz) [inline NEWS.md:18]: Ditto

File: NEWS.md (hunk 1 above)
Comment (mpreiner) [reply NEWS.md]: yes, you are right, fixed.

File: NEWS.md (hunk 2 above)
Comment (mpreiner) [reply NEWS.md]: fixed

**Review by aniemetz — APPROVED** [review]

**Pull request merged**